quantestpy/circuit.py

```diff
@@ -6,41 +6,17 @@
 
```

Comment:
Should we place `circuit.py`, `operator.py` and `state_vector.py` in `quantestpy/assertion`?

Reply:
That is out of scope of this PR. That will be done in https://github.com/QuantestPy/quantestpy/issues/175